scripts/generate-framework-agreement-signature-pages.py

```diff
@@ -63,28 +66,44 @@
 if __name__ == '__main__':
     args = docopt(__doc__)
 
+    if args['<supplier_id_file>']:
+        supplier_ids = get_supplier_ids_from_file(args['<supplier_id_file>'])
+    elif args['--supplier-id']:
+        supplier_ids = args['--supplier-id']
+    else:
+        supplier_ids = None
```

Comment:
I think we should fail noisily if there are no suppliers provided - if we're running this script then we should have some! At the very least let's log that nothing's going to happen.

Reply:
Ah, if you don't supply a supplier ID then it gets all suppliers; that should be documented

Reply:
This is now documented, see https://github.com/alphagov/digitalmarketplace-scripts/pull/335/files#diff-7df3cea3a04365ea67135cd2d5e6d21aR26